lagom-renderers/javadsl/src/main/scala/com/lightbend/lagom/spec/render/model/JavaEnumRender.scala

```diff
@@ -0,0 +1,60 @@
+package com.lightbend.lagom.spec.render.model
+
+import com.lightbend.lagom.spec.model.{ CustomEnum, Service }
+
+/**
+ *
+ */
+object JavaEnumRender {
+  val jacksonImports = "import com.fasterxml.jackson.annotation.*;"
+
+  def packageDeclaration(service: Service): String = s"package ${service.`package`};"
+
+  private def enumValues(customEnum: CustomEnum) = {
+    customEnum.values.map(v => s"""    ${v.toUpperCase.replaceAll("-", "_")}("$v")""").mkString(",\n")
+  }
+
+  private def enumDefinition(customEnum: CustomEnum) = {
+    s"""public enum ${customEnum.className} {
+       |${enumValues(customEnum)};
+       |
+      |    private final String value;
+       |
+      |    ${customEnum.className}(String value) {
+       |      this.value = value;
+       |    }
+       |
+      |    @com.fasterxml.jackson.annotation.JsonValue
+       |    public String getValue() {
+       |      return value;
+       |    }
+       |
+      |    @Override
+       |    public String toString() {
+       |      return String.valueOf(value);
```

Comment:
Several parts of the code templates are reused from other providers and/or the hackathon. I don't think this should remain.

This code overrides `toString` of the enum so that the internal value (the actual string parsed from JSON) is used instead of the enum ID. So if the enum is: 

```
public enum Foo{ 
  Bar("bar_value");
  private Foo(String value){ this.value = value;}
...
```
then `toString` will print `bar_value`. It could make sense if the enum ids were random stuff but I'm not sure it makes sense since we're generating the equivalents.

On second thought, it could make make sense if/when we want to support `UNKNOWN` as a possible enum value (for additional properties). See #16.